cvat/apps/dataset_manager/annotation.py

```diff
@@ -988,3 +1004,6 @@ def _unite_objects(obj0, obj1):
         track["shapes"] = list(sorted(shapes.values(), key=lambda shape: shape["frame"]))
 
         return track
+
+    def clear_frames(self, frames: Container[int]):
+        raise AssertionError("This function is not supported for tracks")
```

Comment:
`NotImplementedError` can leave some hope that it can be implemented at some point later. It's not what is supposed to happen here; it's intentionally disabled here.